argo-configuration/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
```

Comment:
this chart should be removed from this repository.
The only purpose of this repository is to install argo events and argo workflows.
The resources of this chart should be moved to helm-pipelines repository.